### PR TITLE
Show product details

### PR DESCRIPTION
**Pull request by trinhdvt**
## Related Tickets
- [#46905](https://edu-redmine.sun-asterisk.vn/issues/46905)

## WHAT (optional)
- Fix previous PR's comment
- Create show product details page

## HOW
- When user click on product in hompage, browser will navigate to product page, find product by `slug` then show its details

## Evidence (Screenshot or Video)

Ảnh chụp Màn hình 2022-03-15 lúc 11 44 05

Ảnh chụp Màn hình 2022-03-15 lúc 11 40 00

### CODE REVIEW

**Review by lcko1012 — APPROVED** [review]
ok

**Comment by trinhdvt** [comment]
ready

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    @product = Product.find_by slug: slug
+    return if @product
+
+    solve_not_found
+  end
+
+  def solve_not_found
+    flash[:danger] = t ".not_found"
+    redirect_to root_url
+  end
```
Comment (ghost) [inline app/controllers/products_controller.rb]: cái này gọi 1 lần ko cần tách hàm riêng e

File: app/views/products/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<div class="row white_bg">
+  <div class="col-md-4 product_image">
+    <div style="background-image: url('<%= @product.image %>');">
```
Comment (ghost) [inline app/views/products/show.html.erb]:
```suggestion
    <div style="background-image: url("<%= @product.image %>");">
```

**Review by ghost — COMMENTED** [review]
cmt nhỏ

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**